src/cc/usdt/usdt.cc

```diff
@@ -393,10 +394,15 @@ extern "C" {
 void *bcc_usdt_new_frompid(int pid, const char *path) {
   USDT::Context *ctx;
 
-  if (!path)
+  if (!path) {
     ctx = new USDT::Context(pid);
-  else
+  } else {
+    struct stat buffer;
+    if ((strlen(path) >= 1 && path[0] != '/' ) || stat(path, &buffer) == -1) {
```

Comment:
Could you put the error messages here? You can see from `libbpf.c` and `usdt.cc` where a lot of errors are printed out simply with `fprintf(stderr, ...)`. This way, in python, you do not need to check `startwith('/'` or `open(path)` any more, you can just throw an exception since the detailed errors and hints have been printed out.

If you added the error messages in `bcc_usdt_new_frompid`, C++ and lua will be automatically covered.